### PR TITLE
Playback Speed Range increased to 3*

### PR DESCRIPTION
**Pull request by Sahilsingh0808**
**Playback Speed Range increased**: The range of playback speed is set from 2.00 to 3.00 to give the user more freedom to choose their podcast streaming speed. The speed steps' remains 0.25 and the number of speed blocks are increased from 6 to 10.
Screenshot_20220419_194841

Screenshot_20220419_194939

**Flutter Packages updated**: The packages from pub.dev have been updated to their latest version. These include a few packages that turned a few versions outdated in the meantime.

### CODE REVIEW

**Review by roeierez — APPROVED** [review]

**Review by ademar111190 — CHANGES_REQUESTED** [review]

File: pubspec.yaml [hunk 1]
```diff
@@ -101,7 +101,7 @@ dependency_overrides:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  build_runner: ^2.1.7
+  build_runner: ^2.1.10
```
[the remark this answers is not in the document]
Comment (Sahilsingh0808) [reply pubspec.yaml]: I have removed the updates from the yaml file now.
Thanks

[the remark this answers is not in the document]
Comment (Sahilsingh0808) [reply pubspec.yaml]: I have kept the yaml file as par with the current version of the original breezmobile repo.

[the remark this answers is not in the document]
Comment (ademar111190) [reply pubspec.yaml]: It is still in the diff for me

[the remark this answers is not in the document]
Comment (Sahilsingh0808) [reply pubspec.yaml]: Currently, my yaml file in the pull request is the same as the yaml file in the breezmobile repo. Should it be changed?

[the remark this answers is not in the document]
Comment (ademar111190) [reply pubspec.yaml]: It is not the same @Sahilsingh0808 take a look at the Files Changed tab.

[the remark this answers is not in the document]
Comment (Sahilsingh0808) [reply pubspec.yaml]: Shall I delete the file: pubspec.yaml from the pull request?

[the remark this answers is not in the document]
Comment (kingonly) [reply pubspec.yaml]: I think it's better to delete it, yes

[the remark this answers is not in the document]
Comment (Sahilsingh0808) [reply pubspec.yaml]: Yes its done now

**Pull request merged**